### PR TITLE
Fix some mistakes

### PR DESCRIPTION
**Pull request by flash2048**
Fix some mistakes

### CODE REVIEW

File: src/Raven.Server/Documents/Queries/QueryBuilder.cs [hunk 1]
```diff
@@ -735,7 +735,7 @@ LuceneTermType GetTermType(string termValue)
             AssertValueIsString(fieldName, wktValue.Type);
 
             SpatialUnits? spatialUnits = null;
-            if (expression.Arguments.Count == 2)
+            if (expression.Arguments.Count == 4)
```
Comment (ayende) [inline src/Raven.Server/Documents/Queries/QueryBuilder.cs:738]: What it this change for?

Comment (ayende) [reply src/Raven.Server/Documents/Queries/QueryBuilder.cs:738]: I can see from the call that it is needed, but can you send a test as well?

Comment (ppekrol) [reply src/Raven.Server/Documents/Queries/QueryBuilder.cs:738]: `spatial.wkt` only accepts one argument at the moment. This was copied by accident. Check: https://github.com/ravendb/ravendb/blob/0d76f0311f99b6dee5a81eaf1ec8534049ae008c/src/Raven.Server/Documents/Queries/QueryValidator.cs#L25

We should either:
1. expose it in RQL and Client API
2. remove this condition entirely

**Review by ppekrol — CHANGES_REQUESTED** [review]
Check my comment in code.

File: src/Raven.Server/Documents/Queries/QueryBuilder.cs (hunk 1 above)
Comment (ayende) [reply src/Raven.Server/Documents/Queries/QueryBuilder.cs:738]: Let's remove it for now and create an issue for the future

**Pull request closed without merging**